tests/ignite/contrib/metrics/regression/test_wave_hedges_distance.py

```diff
@@ -16,7 +16,7 @@ def test_wrong_input_shapes():
         m.update((torch.rand(4), torch.rand(4, 1)))
 
     with pytest.raises(ValueError, match=r"Input data shapes should be the same, but given"):
-        m.update((torch.rand(4, 1), torch.rand(4,)))
+        m.update((torch.rand(4, 1), torch.rand(4,),))
```

Comment:
It seems like one of the code-styling libraries (isort, black, flake8) added this

Reply:
This seems strange...

Reply:
Yes, it's the second time this happens! check this #1869 at the second commit i deleted it, i thought i added it by mistake, but turns out i didn't!